components/Prize/PrizeExamples.jsx

```diff
@@ -0,0 +1,58 @@
+'use strict'
```

Comment:
`'use strict'` can be deleted (it's only needed in `.cjsx` files, which we are not using)